psql2mysql/tests/test_dbwrapper.py

```diff
@@ -82,3 +82,36 @@ def test_scanTableForLongTexts(self, mock_query_long_text_rows,
         wrong_item = result[0]
         self.assertEqual("text", wrong_item["column"])
         self.assertEqual("id=3", wrong_item["primary"][0])
+
+    def test_chunked_write_table_rows(self):
+        mock_table = mock.MagicMock()
+        mock_connection = mock.MagicMock()
+        mock_rows = mock.MagicMock()
+        mock_rows.fetchmany.side_effect = [['one response'],
+                                           ['second response'], []]
+
+        self.db_wrapper.chunk_size = 1
+        self.db_wrapper.connection = mock_connection
+        self.db_wrapper.writeTableRows(mock_table, mock_rows)
+
+        self.assertTrue(mock_rows.fetchmany.called)
+        self.assertEqual(mock_rows.fetchmany.call_count, 3)
```

Comment:
IMO, just checking for call_count is fine rather than both .called and then asserting the right number of calls.

Reply:
Maybe on the fetchall case. I wanted to be sure/test the fetchmany was iterating/looping through all results. Thus the call_count check.